### PR TITLE
Making DataOcean parent model

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/models/ratu_models.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 from django.db import models
```
Comment (Iva-khar) [inline data_ocean/models/ratu_models.py:1]: Is it still need?

File: data_ocean/models/rfop_models.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 from django.db import models
```
Comment (Iva-khar) [inline data_ocean/models/rfop_models.py:1]: the same

File: data_ocean/models/ruo_models.py [hunk 3]
```diff
@@ -1,14 +1,15 @@
 from django.db import models
```
Comment (Iva-khar) [inline data_ocean/models/ruo_models.py:1]: the same

File: data_ocean/models/ratu_models.py (hunk 1 above)
Comment (Iva-khar) [inline data_ocean/models/ratu_models.py:1]: Is it still need?

File: data_ocean/models/rfop_models.py (hunk 2 above)
Comment (Iva-khar) [inline data_ocean/models/rfop_models.py:1]: the same

File: data_ocean/models/ruo_models.py (hunk 3 above)
Comment (Iva-khar) [inline data_ocean/models/ruo_models.py:1]: the same

File: data_ocean/models/ratu_models.py (hunk 1 above)
Comment (IegorSoboliev) [reply data_ocean/models/ratu_models.py:1]: Yes, because we still use functions of models

File: data_ocean/models/rfop_models.py (hunk 2 above)
Comment (IegorSoboliev) [reply data_ocean/models/rfop_models.py:1]: Yes, because we still use functions of models

File: data_ocean/models/ruo_models.py (hunk 3 above)
Comment (IegorSoboliev) [reply data_ocean/models/ruo_models.py:1]: Yes, because we still use functions of models

**Pull request merged**